packages/astro/src/assets/utils/etag.ts

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Should we also copy the licence file? 

EDIT: they don't have a licence file, so maybe we could link: https://github.com/tjwebb/fnv-plus#license